src/components/pages/MyCases/ReviewCaseForm.js

```diff
@@ -42,6 +42,9 @@ const ReviewCaseForm = props => {
       .catch(err => {
         console.log(err);
       });
+    console.log('I have tried to submit');
+    console.log(editedFormValues);
+    console.log(evt);
```

Comment:
It looks like these three console logs were added. It would be best to remove them before merging this.

Reply:
I agree with Josh here and removing these console logs would be a good idea before merging.

Reply:
Oops, I am not 100% how this got there! It definitely wasn't there when I initially pushed the PR, but seems to have leaked in while I was working on other tickets. Super grateful you caught this!